src/components/organisms/pupil/OakPupilJourneyHeader/OakPupilJourneyHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { OakFlex, OakHeading, OakIconProps } from "../../../atoms";
+import {
+  OakBulletList,
+  OakBulletListProps,
+  OakHandDrawnCardWithIcon,
+} from "../../../molecules";
+
+export type OakPupilJourneyHeaderProps = {
+  title: string;
+  iconName: OakIconProps["iconName"];
+  alt?: OakIconProps["alt"];
+  iconBackground?: "primary" | "secondary";
+} & OakBulletListProps;
+
+/**
+ * This component is the header for the pupil journey;
+ *
+ * the icon, title and list of items are passed as props and change change depending on which page it is called
+ *
+ *
+ */
+export const OakPupilJourneyHeader = ({
+  iconBackground = "primary",
+  ...props
+}: OakPupilJourneyHeaderProps) => {
+  return (
+    <OakFlex $flexDirection={"row"} $gap={"space-between-m"} {...props}>
```

Comment:
I'm not sure spreading props is wise here. There's a lot of unused values here which could end up causing unexpected behaviours.

Reply:
ive changed it to flexProps; i thin its useful to be able to pass props down to the parent flex container at the page level

Reply:
It's improved but I still don't like it as an interface design. 2 props are exposed but are actually hardcoded (violates only expose props which are variable) and it's ambiguous as to what all of those generic flex props control. Instead pick the one or two props that you think the user actually needs to adjust at page level and keep the rest internal.

Reply:
ok removed